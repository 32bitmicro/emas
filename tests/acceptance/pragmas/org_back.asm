.org 10
.word 1
.org 10
