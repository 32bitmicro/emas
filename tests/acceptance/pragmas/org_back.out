acceptance/pragmas/org_back.asm:3:8: Cannot move location pointer backwards by -1 words
